Util/+IMOS/get.m

```diff
@@ -0,0 +1,49 @@
+function [fcell] = get(icell, fieldname)
+% function [fcell] = get(icell,fieldname)
```

Comment:
get is a pretty generic name

Reply:
Yeah, correct name would be `get_fieldname_from_all_items_in_cell_returning_empty_if_not_found`  hehe.

Maybe `IMOS.get_from` or `IMOS.get_all` or `IMOS.get_all_from` is better, but I went with the more high-level generic name because the function name is the access pattern - "to get" - but works as a "get all fieldnames" by the nature of the argument, even though it only operates on cells. 

Finally, when reading the code the intent is clear:

```
var_names = IMOS.get(sample_data.variables,"name")
```

Otherwise,
```
var_names = IMOS.get_...(sample_data.variables,"name")
```

sounds repetitive when typing and reading. since I usually already know what I want (when typing), and already got the clue from the variable name that the result is more than one item and that the first argument is multiple ( when reading).